app/services/google/scrape_service.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Google
+  class ScrapeService
+    def initialize(keyword_id:, search_service: Google::SearchService.new, parse_service: Google::ParseService.new)
+      @keyword_id = keyword_id
+      @search_service = search_service
+      @parse_service = parse_service
+    end
+
+    def call!
+      # TODO(Goose97): update failed status once we integrate with background job
+      begin
```

Comment:
just a small thing, we do not need the `begin` here @Goose97 so you can delete it